emacs-everywhere.el

```diff
@@ -330,12 +349,14 @@ Never paste content when ABORT is non-nil."
         (when (and (frame-parameter nil 'emacs-everywhere-app)
                    emacs-everywhere-paste-command
                    (not abort))
+	  (sleep-for 0.1)
```

Comment:
Why is this `sleep-for` here? If it's only needed for windows, please windows-gate it.

Reply:
That's my mistake. Forgot to delete that which was intended to solve the instability but found not helpful later.